docusaurus.config.js

```diff
@@ -25,7 +25,7 @@ const config = {
   // to replace "en" with "zh-Hans".
   i18n: {
     defaultLocale: "en",
-    locales: ["en"],
+    locales: ["en"/*, "ru", "zh"*/],
```

Comment:
Simple question but the Russian and Chinese are just commented out for now until the translation is ready?

Reply:
Yes, as discussed in our conversation with Laurel on Slack.